### ISSUE TITLE
Create funds as a sponsor (with an implementation)

### ISSUE THREAD

**Issue by maxvisser** [teamforus/general#437]
## Description

All sponsor who had an implementation with Forus still need to contact Forus again for creating new funds. A sponsor who has an implementation is familiar with the system and need less training/implementation than a completely new user.

## Desired result
If a implementation (a webshop) is configured by Forus, funds created by implementation owner will be automatically added to its implementation. Funds created will be configured with implementation and fund details and therefor can skip the waiting state.

Funds created by organization who do not have any implementation (are not a owner of a implementation which is configured by Forus) should still wait on Forus to respond.

## Needed changes
- Forus sets up 'implementation' -> configures bunq key in implementation
	- this is currently done per fund
- Implementation should have organization 'owner' 
    - there is currently no owner, CMS settings are shown when your fund is attached to an implementation
- If a fund is created by implementation owner. It can gets automatically a fund config for that implementation.
- eligible key will set by fund name (sluggified example fund name Meedoenregeling 2020 would become meedoenregeling_2020_eligible)
- record_type for fund key should be automatically created -> translation key is Recht op 'fund_name'
- fund key will be automatically added to criteria (a fund should never be able to have no criteria)
   - this is currently the already the case should check if it creates the record type aswell though.
- children_nth criteria is currently aswell automatically being added. This can be removed

## Results from call:
Part 1
* Define interface changes to auto approved fund
  * New user
  * Make an organization
  * Getting verified (approved with flags)
Create funds from preset (if approved) with auto approval
Define default preset for different fund types
Bunq account
Create funds from presets
Start funds from presets

Type of fund: Without formula
@maxvisser define default fund_config and formula

**Comment by jamalv** [comment]
Discussed this issue; the goal is to focus on the "default" configuration, not the exemptions.

@maartenfv could you please plan time (2 hours?) for valentin and max to refine this issue, and then for development? Maybe for dimitri? With this issue he will learn a lot about the platform I think :)

**Comment by maartenfv** [comment]
Planned it in. 

Consideration = We hope to migrate to BNG before 1 st of jan 2022.

**Comment by jamalv** [comment]
currently not planned in, no project, superseded by: https://roadmap.forus.io/c/67-zelfstandige-aanmelding-van-sponsoren
